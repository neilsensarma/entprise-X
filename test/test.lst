NP_689699.2 343 TRP ARG
NP_056473.2 300 ILE VAL
NP_001153656.1 4 SER THR
NP_001153656.1 413 SER LEU
NP_001153656.1 452 ARG HIS
NP_001153656.1 452 ARG PRO
NP_001153656.1 476 SER PRO
NP_940978.2 1775 ALA SER
NP_001123517.1 112 GLY ARG
NP_689414.1 5 ALA THR
NP_689414.1 757 CYS SER
NP_689414.1 757 CYS ARG
NP_001140157.1 45 PRO LEU
NP_114127.3 522 THR MET
NP_001164159.1 72 PHE LEU
NP_001164158.1 168 GLY VAL
NP_001164160.1 499 SER LEU
NP_277023.1 63 VAL ALA
NP_277023.1 63 VAL GLY
NP_277023.1 59 ARG TRP
NP_001186716.1 229 ARG HIS
NP_001185924.1 414 GLY GLY
NP_001185924.1 230 ASN LYS
NP_848640.1 141 MET THR
NP_001073953.1 628 GLY CYS
NP_001073953.1 404 LYS GLU
NP_001073953.1 363 ILE VAL
NP_001073953.1 363 ILE PHE
NP_055453.2 1349 ILE LEU
NP_003811.2 17 LYS ARG
NP_003811.2 241 VAL ILE
NP_258428.2 518 MET THR
NP_955533.2 533 SER PRO
NP_055263.2 370 HIS TYR
NP_001400.3 1413 HIS TYR
NP_001400.3 1392 TRP ARG
NP_001400.3 916 ARG LEU
NP_001400.3 688 LEU PRO
NP_001400.3 131 SER GLY
NP_055519.1 617 ALA ASP
NP_997239.2 53 THR ARG
NP_056372.1 1539 SER PRO
NP_056372.1 1298 GLU ASP
NP_001019769.1 96 PRO THR
NP_997253.2 209 ARG HIS
NP_001036129.1 414 ALA THR
NP_001036129.1 238 THR SER
NP_078930.3 10 ARG TRP
NP_619642.2 347 LYS GLU
NP_001265576.1 146 ALA SER
NP_056030.1 1082 GLN HIS
NP_068835.1 21 THR MET
NP_001036147.1 324 ALA ASP
NP_001206.2 58 ARG TRP
NP_997303.2 261 ARG GLN
NP_001124396.2 579 PRO LEU
NP_006601.2 665 GLN LYS
NP_006601.2 371 ASP TYR
NP_631947.1 120 ASP GLY
NP_066969.1 83 GLU ASP
NP_065831.1 182 GLY SER
NP_065831.1 650 ALA THR
NP_060908.1 293 PHE LEU
NP_940947.3 37 LEU PHE
NP_001243888.1 176 GLU GLY
NP_068752.2 99 LYS ASN
NP_068752.2 167 LYS GLU
NP_001057.1 196 MET ARG
NP_001057.1 232 GLU LYS
NP_001096640.1 47 SER PRO
NP_001096639.1 125 CYS PHE
NP_075389.1 138 CYS ARG
NP_075389.1 138 CYS GLY
NP_075389.1 252 GLN GLU
NP_075389.1 310 MET VAL
NP_075389.1 372 ALA GLY
NP_075389.1 430 PHE LEU
NP_075389.1 430 PHE LEU
NP_075389.1 464 PRO LEU
NP_075390.1 464 PRO LEU
NP_001006626.1 105 ALA GLY
NP_056950.2 283 ASP GLU
NP_056950.2 283 ASP GLU
NP_001007258.1 501 THR PRO
NP_001129082.1 198 GLN ARG
NP_443161.1 356 ILE MET
NP_443161.1 651 GLY ARG
NP_443161.1 1262 MET VAL
NP_443161.1 1325 ILE VAL
NP_056933.2 114 ASP ASN
NP_056933.2 177 GLN ARG
NP_127463.2 138 GLN ARG
NP_001264983.1 28 ALA VAL
NP_079034.3 140 ARG GLN
NP_055979.2 32 ILE THR
NP_055979.2 249 ARG SER
NP_055816.2 235 ARG LEU
NP_055816.2 2007 ASP GLU
NP_055816.2 2360 ASN ASP
NP_001036169.1 83 ARG GLY
