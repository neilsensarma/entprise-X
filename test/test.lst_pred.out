NP_689699.2 343 -0.0101158
NP_056473.2 300 0.212441
NP_001153656.1 4 -0.0128734
NP_001153656.1 413 0.00822947
NP_001153656.1 452 -0.0311933
NP_001153656.1 452 -0.0311933
NP_001153656.1 476 -0.00469904
NP_940978.2 1775 0.479739
NP_001123517.1 112 0.024088
NP_689414.1 5 0.196881
NP_689414.1 757 0.214649
NP_689414.1 757 0.214649
NP_001140157.1 45 -0.0230098
NP_114127.3 522 0.11625
NP_001164159.1 72 0.0303443
NP_001164158.1 168 0.0606155
NP_001164160.1 499 0.0838906
NP_277023.1 63 0.111028
NP_277023.1 63 0.111028
NP_277023.1 59 0.133961
NP_848640.1 141 0.0512228
NP_001073953.1 628 0.0206416
NP_001073953.1 404 0.0715085
NP_001073953.1 363 0.09365
NP_001073953.1 363 0.09365
NP_055453.2 1349 0.221896
NP_003811.2 17 0.802226
NP_003811.2 241 0.751659
NP_258428.2 518 0.845157
NP_955533.2 533 0.701984
NP_055263.2 370 0.383711
NP_001400.3 1413 0.0641104
NP_001400.3 1392 0.0161345
NP_001400.3 916 -0.00511574
NP_001400.3 688 0.0749718
NP_001400.3 131 0.00745222
NP_055519.1 617 0.0633631
NP_997239.2 53 0.104575
NP_056372.1 1539 0.337753
NP_056372.1 1298 0.403339
NP_001019769.1 96 0.0266607
NP_997253.2 209 -0.00290062
NP_001036129.1 414 0.285493
NP_001036129.1 238 0.362549
NP_078930.3 10 -0.0133222
NP_619642.2 347 0.400491
NP_056030.1 1082 0.397639
NP_068835.1 21 0.679495
NP_001206.2 58 0.486226
NP_997303.2 261 0.220519
NP_001124396.2 579 -0.0147166
NP_006601.2 665 0.215709
NP_006601.2 371 0.396711
NP_631947.1 120 0.363984
NP_066969.1 83 0.127881
NP_065831.1 182 -0.000229411
NP_065831.1 650 0.0131889
NP_060908.1 293 0.0108193
NP_940947.3 37 0.0271788
NP_068752.2 99 0.0282475
NP_068752.2 167 0.0295053
NP_001057.1 196 0.652627
NP_001057.1 232 0.623622
NP_001096640.1 47 0.0200616
NP_001096639.1 125 0.0131363
NP_075389.1 138 0.0229067
NP_075389.1 138 0.0229067
NP_075389.1 252 0.0404538
NP_075389.1 310 0.0579801
NP_075389.1 372 0.0136887
NP_075389.1 430 0.00780128
NP_075389.1 430 0.00780128
NP_075389.1 464 -0.0432185
NP_075390.1 464 -0.0658652
NP_001006626.1 105 0.41678
NP_056950.2 283 0.557947
NP_056950.2 283 0.557947
NP_001007258.1 501 0.588854
NP_001129082.1 198 0.261323
NP_443161.1 356 0.11225
NP_443161.1 651 0.0314787
NP_443161.1 1262 0.128281
NP_443161.1 1325 0.135772
NP_056933.2 114 0.108934
NP_056933.2 177 0.0705332
NP_079034.3 140 0.0487869
NP_055979.2 32 0.0611309
NP_055979.2 249 -0.0180838
NP_055816.2 235 0.480088
NP_055816.2 2007 0.674076
NP_055816.2 2360 0.683963
NP_001036169.1 83 0.521804
